### ISSUE TITLE
More information in raspberrypi.org resources, please.

### ISSUE THREAD

**Issue by LonnieRisckle** [raspberrypilearning/getting-started-with-git#16]
The page for Apple Macs: https://www.raspberrypi.org/learning/getting-started-with-git/mac-install/ assumes that Ruby is already installed.

Would you please add a line saying 'If you don't have Ruby, you can get it from **here**?

This will allow people to pick it up from a trusted source, rather than searching round the Internet and possibly ending up with malware from a fake site.

**Comment by lawsie** [comment]
@rik-cross @MarcScott This one's for you :)

**Comment by MarcScott** [comment]
I'm pretty sure that Ruby has been included in OSX and MacOS since Tiger (2005). What version of the OS are you running @LonnieRisckle

**Comment by LonnieRisckle** [comment]
OK, either something strange is happening, or I'm a complete pillock. (Yes, it's probably me.)

Yesterday I got a 'command not found' error when I cut and pasted the command, but today it worked. (I went in today to get the exact error message.)

In my defence, there's no sign of Ruby in Spotlight, AppStore, Launchpad or Finder, so I couldn't think of anywhere else to look.

Thanks for your help!
